addOns/httpsInfo/src/main/java/org/zaproxy/zap/extension/httpsinfo/HttpsInfoOutputPanel.java

```diff
@@ -118,14 +118,15 @@ private void doThreadedTasks() {
 			public void run() {
 				try {
 					initSession(getTarget());
+					setDvEng(getSession());
 				} catch (DVException e) {
-					String warnMsg = Constant.messages.getString("httpsinfo.init.warning", getTarget().toString(),
-							e.getCause()); 
+					String warnMsg = Constant.messages.getString(
```

Comment:
This change does not seem to be necessary.

Reply:
@thc202 improving code-style never seems unnecessary.

Reply:
Yes, but the problem with these style/formatting changes is that what's an improvement to someone is not to another, so, following contribution will change those lines to match the style it prefers most (that's why it's better to leave existing code as is). I know that this is not ideal but we are working on normalising the format/style throughout the ZAP repos, zap-extensions and zaproxy are the last ones (since they required/require some more work).